velox/exec/ExchangeClient.h

```diff
@@ -61,6 +62,8 @@ class ExchangeClient {
   void close();
 
   // Returns runtime statistics aggregated across all of the exchange sources.
+  // Runtime statistics are expected include background CPU time in a metric
```

Comment:
typo: expected -> expected to

please, add a period at the end of the sentence.

Reply:
Fixed.